R/teal_data-get_code.R

```diff
@@ -0,0 +1,50 @@
+#' Get Code from `teal_data`
+#'
+#' Get code from [`teal_data`]. The extraction of the code limited to specific objects can be performed with the usage
+#' of `names` parameter. If input `teal_data@verified` is `FALSE` the returned code is pre-pended with a warning
+#' stating that the object was not verified.
+#'
+#' @param x [`teal_data`] object.
+#' @param names (`character`) vector of object names to return the code for.
+#' @param deparse (`logical`) whether return deparsed form of a call.
```

Comment:
```suggestion
#' @param deparse (`logical`) flag specifying whether to return code as `language` or `expression`.
```

Reply:
@chlebowa for `deparse = TRUE` it's returned as `character` and for `deparse = FALSE` this is `expression` (`parse(text = teal_data@code)`). So this is consistent with `teal.code::get_code`. With your suggestion it means like `language` for one value of `deparse` or `expression` for the other value of deparse.

Maybe we can rephrase to 
`#' @param deparse (\code{logical}) flag specifying whether to return code as \code{character} (\code{deparse = TRUE}) or as \code{expression} (\code{deparse = FALSE}).`

Reply:
My phrasing doesn't state which one is returned for which value.

Invert it, I don't care.

Reply:
It doesnt but it specifies cases that it's gonna be either language or expression, where actually we return character or expression. That what I meant. Will update so it's clear

Reply:
You're right. Sorry, I misread you comment. I had written it as `character` or `language` first and then changed `character` to `expression` instead of changing `language`.

Reply:
8cd6d4fc3f95792bed913b64674ace9abd8cf1a5